src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/resources/PrisonResource.kt

```diff
@@ -68,7 +68,7 @@ class PrisonResource(
   )
 
   @GetMapping(
-    value = ["/prison/prisoner/{prisonNumber}/image", "/prisoner/{prisonNumber}/image"],
+    value = ["/prison/prisoner/{prisonNumber}/image", "/prisoners/{prisonNumber}/image"],
```

Comment:
think we should move WPIP UI to `/prisoners/{prisonNumber}/image` and remove `/prison/prisoner/{prisonNumber}/image`